src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+# Genetic Descriptor
+
+The extension was primarily developped by Cyril Pernet and Clara Moreau with contributions from Tom Nichols and Jessica Turner.
+
+The goal of the genetic descriptor is to link imaging and genetic data by providing minimal information about i) where to find genetic information associated with the imaging data and ii) what type of genetic information is available. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data.
+
+## dataset_description.json
+
+Two additional keys related to the genetic data can be added. The Key `GeneticDataBase` (MANDATORY) links to the name of the database and web address. The key `GeneticDescriptor` (OPTIONAL) refers to the descriptor (e.g. journal article) of the genetic data.
+
+`dataset_description.json` example:
+
+{  
+  "Name": "xxxxx",  
+  "BIDSVersion":  "xxxxx",  
+  "License": "CC0",  
+  "Authors": ["1st author", "2nd author"],  
+  "Funding": "list your funding sources",  
+  "GeneticDataBase": ["Name", "www",],  
+  "GeneticDescriptor": ["data paper", "10.0.2.3/dfjj.10"]  
+}
```

Comment:
For this example, you can use inline code formatting of Markdown. Use Three backticks to start a code paragraph ` ... then type which kind of code is following (JSON) ... then your example ... finally, three closing backticks.

```JSON
`` `JSON
{
"My": "example"
}
`` `
```

Please note that I intentionally left a blank between the backticks to prevent it from rendering and so that you can better see what I mean. See also the GitHub help: https://help.github.com/en/articles/creating-and-highlighting-code-blocks


---

Finally, once the formatting is done, please fill in the "xxx" fields with actual examples, a proper name, version, and database ... of course these don't have to make sense completely, but it'd be a more obvious example.

Reply:
`dataset_description.json` example:  
```JSON   
{  
  "Name": "Human Connectom Project",
  "BIDSVersion":  "1.2.0",
  "License": "CC0",
  "Authors": ["1st author", "2nd author"],
  "Funding": "list your funding sources", 
  "GeneticDataBase": ["Name", "dbGaP Study Accession: phs001364.v1.p1",],
  "GeneticDataBase": ["dbGaP", "phs001364.v1.p1"],
  "GeneticDescriptor": ["The HCP SNP dataset, FancyJOurnal"]
}  
```